### ISSUE TITLE
Docs at http://django-vanilla-views.org/ not up to date

### ISSUE THREAD

**Issue by adamchainz** [encode/django-vanilla-views#72]
I just looked to adding this to a project, I was surprised to read that only Django up to 1.8 is supported on: http://django-vanilla-views.org/#requirements . This isn't the case on the repo, has the documentation build been failing?

**Comment by adamchainz** [comment]
`tox.ini` or `.travis.yml`?

**Comment by carltongibson** [comment]
Ah, yes. My pet hate. 😄 

Alas, the answer is probably _both_. (But we might argue we don't really need `tox` locally for development...)

**Comment by tomchristie** [comment]
Just `.travis.yml` would be fine. I'd be okay with us dropping tox.

**Comment by tomchristie** [comment]
I reckon it's worth us bringing this back up to date and getting it under the encode umbrella. https://github.com/encode/

**Comment by adamchainz** [comment]
Can I do anything to help with move to encode/updating docs?

**Comment by adamchainz** [comment]
Ok so I figured out changing the docs in #75 , #76, and #77. Currently it relies on running the `mkdocs.py` script in the repo (not `mkdocs` the tool, perhaps an early prototype @tomchristie ?) and copying the results over to the `gh-pages` branch as I did in #77. They're up to date now at least.

@tomchristie can you give me access to the repo settings, then I'll move them to HTTPS using the new free Lets Encrypt on github pages :)

**Comment by adamchainz** [comment]
Fair enough, yeah you can't make me admin of a personal repo. And yes, it looks like you might need to go through custom domain setup to find some new DNS records to add, so they can LetsEncrypt.

**Comment by adamchainz** [comment]
I've just updated the docs for the 1.1.0 release.

If you could check "Enforce HTTPS" @tomchristie that'd be neat.

**Comment by adamchainz** [comment]
@tomchristie I appreciate you're busy - perhaps you could add me as collaborator on the repo so I can do this and not ask you in the future?
